Makefile

```diff
@@ -65,13 +66,13 @@ docker_build_master:
 
 .PHONY: docker_push_release
 docker_push_release:
-	@docker login --username $(DOCKERHUB_USERNAME) --password $(DOCKERHUB_PASSWORD)
+	@docker login --username $(QUAY_USERNAME) --password $(QUAY_PASSWORD) quay.io
 	docker push $(DOCKER_LATEST_TAG)
```

Comment:
Why do we push latest and release tag both here for `docker_push_release`?

Reply:
This is to publish an image with the `<release-tag>`, and also to publish an image with the `latest` tag, ie so you can either pull a specific version, or just the latest tagged version